### ISSUE TITLE
Error: Cannot read config file: while running yarn lint

### ISSUE THREAD

**Issue by anksingh-mdsol** [railsware/js-routes#304]
Facing below issue while running yarn lint:

```
yarn lint

Oops! Something went wrong! :(

ESLint: 8.26.0

Error: Cannot read config file:.../node_modules/eslint-config-prettier/@typescript-eslint.js
Error: "prettier/@typescript-eslint" has been merged into "prettier" in eslint-config-prettier 8.0.0. See: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21
```

```
bundle info js-routes
  * js-routes (2.2.4)
        Summary: Brings Rails named routes to javascript
        Homepage: http://github.com/railsware/js-routes
```

**Comment by bogdan** [comment]
I've updated prettier and eslink within the package. Hope it helps. Can you please check if the error is reproducable from master now?

**Comment by anksingh-mdsol** [comment]
Yup, this resolved it!

**Comment by bogdan** [comment]
Released 2.2.5
